util/regexp-assemble/data/942370.data

```diff
@@ -3,22 +3,44 @@
 
 ##!+ i
 
-[\"'`]\s*?\*.+or\W*?[\"'`]\d
-[\"'`]\s*?\*.+xor\W*?[\"'`]\d
-[\"'`]\s*?\*.+div\W*?[\"'`]\d
-[\"'`]\s*?\*.+like\W*?[\"'`]\d
-[\"'`]\s*?\*.+between\W*?[\"'`]\d
-[\"'`]\s*?\*.+and\W*?[\"'`]\d
-[\"'`]\s*?\*.+id\W*?[\"'`]\d
+##!> assemble
+[\"'`]\s*?
+##!=>
+ ##!> assemble
+  ##!> assemble
+  ##!=>
+\*.+
+  ##!=>
+   ##!> assemble
+or
+xor
+div
+like
+between
+and
+id
+   ##!=>
+\W*?[\"'`]\d
+   ##!=>
+    ##!<
+   ##!<
+[^\w\s?]+\s*?[^\w\s]+\s*?[\"'`]
+[^\w\s]+\s*?[\W\d].*?#
+[^\w\s]+\s*?[\W\d].*?--
+   ##!> assemble
+or
+xor
+div
+like
+between
+and
```

Comment:
You can store the list of words that is repeated (or, xor, div, etc.) and reuse it, so you don't have to write it down twice.